AdobeStockImageAdminUi/view/adminhtml/web/js/mediaGallery/grid/columns/image/licenseActions.js

```diff
@@ -101,8 +101,29 @@ define([
                     response.imageDetails.id =  response.imageDetails['adobe_stock'][0].value;
                     response.imageDetails.category =  response.imageDetails['adobe_stock'][3].value;
                     this.image().displayedRecord(response.imageDetails);
-                    this.image().actions().licenseProcess();
-                    this.imageModel().reloadGrid();
+                    $.ajaxSetup({
+                        async: false
```

Comment:
Is there a case when we can license image or request confirmation asynchronously? If not, I think we can set the corresponding requests to be performed asynchronously rather than changing global setting